torch/testing/_internal/common_methods_invocations.py

```diff
@@ -108,7 +108,8 @@ def __init__(self,
                  promotes_integers_to_float=False,  # whether op promotes unary output to float or not
                  sample_inputs_func=None,  # function to generate sample inputs
                  aten_name=None,  # name of the corresponding aten:: operator
-                 supports_sparse=False  # supported for sparse
+                 supports_sparse=False,  # supported for sparse
+                 supports_inplace_on_sparse_uncoalesced=False  # supports in-place variant for Sparse Tensors
```

Comment:
This seems like a reasonable metadata addition. After the other discussion is resolved let's revisit this briefly: we may want to just make this metadata something like "is_linear_map," which is a more general statement this property can be inferred from.

Reply:
Thanks for the comments; will work on the same

Reply:
@mruberry - "is_linear_map" looks a more sensible name; have made the necessary changes